extensions/wikia/ArticleVideo/OoyalaConfigController.class.php

```diff
@@ -0,0 +1,345 @@
+				'fontStyleClass' => 'oo-icon oo-icon-volume-on-ooyala-defualt',
+			],
+			'volumeOff' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'p',
+				'fontStyleClass' => 'oo-icon oo-icon-volume-mute-ooyala-defualt',
+			],
+			'expand' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'i',
+				'fontStyleClass' => 'oo-icon oo-icon-system-fullscreen',
+			],
+			'compress' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'j',
+				'fontStyleClass' => 'oo-icon oo-icon-system-minimizescreen',
+			],
+			'ellipsis' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'f',
+				'fontStyleClass' => 'oo-icon oo-icon-system-menu',
+			],
+			'replay' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'c',
+				'fontStyleClass' => 'oo-icon oo-icon-system-replay',
+			],
+			'share' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'c',
+				'fontStyleClass' => 'oo-icon oo-icon-share',
+			],
+			'cc' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'k',
+				'fontStyleClass' => 'oo-icon oo-icon-cc',
+			],
+			'discovery' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'l',
+				'fontStyleClass' => 'oo-icon oo-icon-discovery-binoculars',
+			],
+			'quality' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'm',
+				'fontStyleClass' => 'oo-icon oo-icon-bitrate',
+			],
+			'setting' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'n',
+				'fontStyleClass' => 'oo-icon oo-icon-system-settings',
+			],
+			'dismiss' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'e',
+				'fontStyleClass' => 'oo-icon oo-icon-system-close',
+			],
+			'toggleOn' => [
+				'fontFamilyName' => 'fontawesome',
+				'fontString' => '',
+				'fontStyleClass' => '',
+			],
+			'toggleOff' => [
+				'fontFamilyName' => 'fontawesome',
+				'fontString' => '',
+				'fontStyleClass' => '',
+			],
+			'left' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'r',
+				'fontStyleClass' => 'oo-icon oo-icon-system-left-arrow',
+			],
+			'right' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 's',
+				'fontStyleClass' => 'oo-icon oo-icon-system-right-arrow',
+			],
+			'learn' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 't',
+				'fontStyleClass' => 'oo-icon oo-icon-system-more-information',
+			],
+			'skip' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'u',
+				'fontStyleClass' => 'oo-icon oo-icon-skip-slick',
+			],
+			'warning' => [
+				'fontFamilyName' => 'fontawesome',
+				'fontString' => '',
+				'fontStyleClass' => '',
+			],
+			'auto' => [
+				'fontFamilyName' => 'ooyala-slick-type',
+				'fontString' => 'd',
+				'fontStyleClass' => 'oo-icon oo-icon-system-auto',
+			],
+		],
+	];
+
+	public function skin() {
```

Comment:
This method doesn't have template that could be rendered. Can we enforce `json` format then?